app.js

```diff
@@ -11,13 +12,18 @@ app.get('/api', (request, response, next) => {
 
 app.get('/api/topics', getAllTopics )
 
+app.get('/api/articles/:article_id', getArticleById)
+
 
 //last endpoint
```

Comment:
Remember to remove comments from your code before submitting a pull request - generally, production code shouldn't have comments.